packages/styleguide/.storybook/main.ts

```diff
@@ -30,8 +31,11 @@ module.exports = {
   },
 
   webpackFinal: (config: any) => {
+    const commonRules = configs.css().module.rules; /* Codecademy configs */
     config.module.rules = config.module.rules.concat(
-      configs.css().module.rules
+      [{...commonRules[0], include: [
+        /node_modules\/@codecademy/
+      ]}, commonRules[1]],
```

Comment:
The commonRules are what we have in our Codecademy webpack-config for CSS. This change restricts the scope of the Codecademy webpack-config to only apply to codecademy node modules. For all other cases, it'll use the built in configuration for storybook. The main takeaway here is that we want to directly import CSS for our storybook, but not Codecademy as it relies on CSS modules.

Reply:
Yeah Storybook+Webpack configuration is always a whole thing. Since we're going to change up & simplify styles anyway, don't worry if the code colors aren't matching up.

https://codecademy.slack.com/archives/C01RE2NGRT9/p1641500446001200

Reply:
To Do: Ticket syntax highlighting for Codebytes and meet with web platform again to understand future plans better.